src/e131.c

```diff
@@ -164,6 +164,29 @@ int e131_multicast_join_iface(int sockfd, const uint16_t universe, const int ifi
   return setsockopt(sockfd, IPPROTO_IP, IP_ADD_MEMBERSHIP, &mreq, sizeof mreq);
 }
 
+/* Join a socket file descriptor to an E1.31 multicast group using a universe and an IP address to bind to */
+extern int e131_multicast_join_ifaddr(int sockfd, const uint16_t universe, const char *ifaddr) {
+  if (universe < 1 || universe > 63999) {
+    errno = EINVAL;
+    return -1;
+  }
+#ifdef _WIN32
+  if (ifindex != 0) {
+    errno = ENOSYS;
+    return -1;
+  }
```

Comment:
This entire check should be removed as it is not necessary here (no `ifindex` to validate).

Reply:
Done! Thanks for catching that.